test/unit/css.test.js

```diff
@@ -19,4 +19,14 @@ suite('CSS', function() {
       widht: ''
     });
   });
+  test('test florin spacing', function () {
+    var mq,
+        mock = $('#mock');
+
+    mq = MathQuill.MathField($('<span></span>').appendTo(mock)[0]);
+    mq.typedText("f'");
+
+    var mqF = $(mq.el()).find('.mq-f');
+    assert.equal(parseFloat(mqF.css('margin-right')), parseFloat(mqF.css('margin-left')) * 2);
+  });
```

Comment:
Can you use something more generic to test like margin-right - margin-left > 0 toBeTruthy